.github/ISSUE_TEMPLATE/bug-template-code.yaml

```diff
@@ -1,7 +1,7 @@
-name: Bug report (code, content or build issues)
-description: Let us know about an issue you experienced.
-title: '[CODE BUG]: '
```

Comment:
Intentional to remove this line?

Reply:
yes

Reply:
labels will serve to filter us, not titles